### PR TITLE
feat(sdk-coin-apt): new coin generation, apt skeleton

### PR DESCRIPTION
**Pull request by baltiyal**
New Aptos SDK Addition : sdk-coin-apt
Addition of Skeleton Code for Apt Coin to support further implementation
TICKET: COIN-2220

### CODE REVIEW

File: modules/sdk-coin-apt/src/apt.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  public getChain(): string {
+    return 'apt';
+  }
+
+  public getFamily(): string {
+    return 'apt';
+  }
+
+  public getFullName(): string {
+    return 'Apt';
```
Comment (alebusse) [inline modules/sdk-coin-apt/src/apt.ts:46]:
```suggestion
    return 'Aptos';
```

File: modules/sdk-coin-apt/src/tapt.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+   */
+  public getChain(): string {
+    return 'tapt';
+  }
+
+  /**
+   * Complete human-readable name of this coin
+   */
+  public getFullName(): string {
+    return 'Testnet Apt';
```
Comment (alebusse) [inline modules/sdk-coin-apt/src/tapt.ts:32]:
```suggestion
    return 'Testnet Aptos';
```

**Review by alebusse — DISMISSED** [review]
Im assuming is Aptos

**Review by sumanmukherjee03 — DISMISSED** [review]
code owner and docker file changes looks good

File: modules/sdk-coin-apt/src/apt.ts (hunk 1 above)
Comment (baltiyal) [reply modules/sdk-coin-apt/src/apt.ts]: Done.

File: modules/sdk-coin-apt/src/tapt.ts (hunk 2 above)
Comment (baltiyal) [reply modules/sdk-coin-apt/src/tapt.ts]: Done.

File: modules/bitgo/test/browser/browser.spec.ts [hunk 3]
```diff
@@ -33,6 +33,8 @@ describe('Coins', () => {
       XrpToken: 1,
       Rune: 1,
       Trune: 1,
+      Apt: 1,
```
Comment (bhavidhingra) [inline modules/bitgo/test/browser/browser.spec.ts:36]: is this change needed?

Comment (baltiyal) [reply modules/bitgo/test/browser/browser.spec.ts:36]: Yes, it is. Otherwise, one of the Browser Test case was failing previously.

File: modules/sdk-coin-apt/test/integration/index.ts
[the remark this answers is not in the document]
Comment (baltiyal) [reply modules/sdk-coin-apt/test/integration/index.ts:1]: As we are adding skeleton for coin and integration test would be surely added later on. So, had a dilemma to add an empty file or not, as both are logically fine being the skeleton PR. So, got reference from this PR : https://github.com/BitGo/BitGoJS/pull/3943/files and finally added an this file as part of skeleton.

**Review by at31416 — APPROVED** [review]

**Review by hrishikeshjain — APPROVED** [review]

File: modules/sdk-coin-apt/src/lib/transaction.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { BaseKey, BaseTransaction } from '@bitgo/sdk-core';
+import { TxData } from './iface';
+
+export class Transaction extends BaseTransaction {
+  canSign(key: BaseKey): boolean {
+    return false;
```
Comment (rakeshgsharma) [inline modules/sdk-coin-apt/src/lib/transaction.ts:6]: should it always be false?

Comment (baltiyal) [reply modules/sdk-coin-apt/src/lib/transaction.ts:6]: By Default in Skeleton PR, its kept as false.
ton : https://github.com/BitGo/BitGoJS/pull/3943/files#diff-1419a46d08351ae9ec9d24697038e25fb70e828a6a00aeb0e79b6574b0a01ccbR6
sui : https://github.com/BitGo/BitGoJS/commit/8ba86b7a10720a14ff1efa9c4616c1f26d27d8e4#diff-f71c74742aabb6007fbb7e7b3586b41105e41530d63ec45bdccd6fc5cb7e1636R6
Handling of actual value would be done in a later PR.

**Review by biwwy0 — APPROVED** [review]
